src/cls/rgw/cls_rgw.cc

```diff
@@ -2964,6 +2965,7 @@ static int usage_iterate_range(cls_method_context_t hctx, uint64_t start, uint64
 
 
     if (i == num_keys - 1) {
+      *truncated = more;
```

Comment:
@tianshan What if no keys are found.
ie `uint32_t num_keys = keys.size();` is `0`

Reply:
truncated default is false